seaplane-cli/src/cli/cmds/metadata/set.rs

```diff
@@ -52,6 +52,8 @@ impl CliCommand for SeaplaneMetadataSet {
     }
 
     fn update_ctx(&self, matches: &ArgMatches, ctx: &mut Ctx) -> Result<()> {
+        // NOTE: MetadataCtx::from_md_set is inpure and tries to read from STDIN if the value is
```

Comment:
```suggestion
        // NOTE: MetadataCtx::from_md_set is impure and tries to read from STDIN if the value is
```

(sorry, sorry, I can't help myself! 😓 )

Reply:
😅